data/tutorials/language/1ms_00_modules.md

```diff
@@ -385,11 +385,17 @@ module Unit :
 
 The OCaml compiler tool chain can be used to dump an `.ml` file's default interface.
 ```shell
-$ ocamlc -c -i cairo.ml
+$ ocamlc -i cairo.ml
 val message : string
 val hello : unit -> unit
 ```
 
+You can also use Anil Madhavapeddy [`ocaml-print-intf`](https://github.com/avsm/ocaml-print-intf) tool to do the same. You have to install it using `opam install ocaml-print-intf`. You can either:
+* Call it on a `.cmi` file (Compiled ML Interface): `ocaml-print-intf cairo.cmi`.
+* Call it using dune: `dune exec -- ocaml-print-intf cairo.ml`
```

Comment:
```suggestion
* Call it using Dune: `dune exec -- ocaml-print-intf cairo.ml`
```
...or if you literally mean call it with the command `dune` instead of the build system Dune, put it in monospace: 
Call it using `dune`: `dune exec -- ocaml-print-intf cairo.ml`